Microsoft.ML.sln

```diff
@@ -139,6 +139,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.DnnImageFeatur
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.DnnImageFeaturizer.ResNet101", "src\Microsoft.ML.DnnImageFeaturizer.ResNet101\Microsoft.ML.DnnImageFeaturizer.ResNet101.csproj", "{DB7CEB5E-8BE6-48A7-87BE-B91D9AE96F71}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.ML.Entrypoints", "src\Microsoft.ML.Entrypoints\Microsoft.ML.Entrypoints.csproj", "{7504D46F-E4B3-43CB-9B1C-82F3131F1C99}"
```

Comment:
`EntryPoints` is the existing capitalization for this term. We should be consistent everywhere.

Reply:
Ahh good catch.

---
In reply to: [240801210](https://github.com/dotnet/machinelearning/pull/1859#discussion_r240801210) [](ancestors = 240801210)